introduction.md

```diff
@@ -1 +1,2 @@
-This is an introduction to dojo-rules.  Best couse looks to be JavaScript.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?